neuralmonkey/learning_utils.py

```diff
@@ -209,6 +209,13 @@ def training_loop(tf_manager: TensorFlowManager,
                                         tf_manager.best_score_batch),
                                 color='blue')
 
+                            # store also graph parts
+                            all_coders = set.union(
+                                *[rnr.all_coders for rnr in runners])
```

Comment:
To jenom rozebere ten list na `*args` toho unionu. Nevím ale, jesti se v takovémhle případě nepíše ten generátor do obyčejných závorek.